File: PRAK504-2210817120013-Fathiah_Nuraisyah_Radam.py
def reverse(a):
    x = 0
    while (int(a) != 0):
        b = int(a) % 10
        x = x * 10 + b
        a = int(a) / 10
    return x
A, B =input().split()
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: PRAK504-2210817120013-Fathiah_Nuraisyah_Radam.c
#include <stdio.h>
int reverse(int a){
    int b, 
    x = 0;
    while (a!=0){
        b = a % 10;
        x = x * 10 + b;
        a/=10;}
    return x;}
int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}